server/routes/temporary-accommodation/index.ts

```diff
@@ -11,11 +11,12 @@ import manageRoutes from './manage'
 export default function routes(controllers: Controllers, services: Services): Router {
   const router = Router()
 
-  const { dashboardController } = controllers
+  const { dashboardController, staticController } = controllers
 
   const { get } = actions(router, services.auditService)
 
   get('/', dashboardController.index(), { auditEvent: 'VIEW_DASHBOARD' })
+  get('/cookies', staticController.cookies(), { auditEvent: 'VIEW_COOKIES' })
```

Comment:
@bryangaledxw After the change to cookies to the `path` setup, should I be using it as a reference here too?

Reply:
Yep, so `/cookies` becomes `paths.static.cookies.pattern`

Reply:
Okay all done 👍 Hopefully included paths correctly, rather than calling it staticPaths.

Reply:
Cool, cheers, looks good. Strictly speaking I should have done he same for the dashboard!